pentaho-js/src/test/javascript/cdf-legacy/visualization-spec.js

```diff
@@ -21,7 +21,7 @@ describe("The VisualizationAPI Component #", function() {
 
   myDashboard.addParameter('optionParameter', "value");
 
-  var visualizationAPIComponent = window.VisualizationAPIComponent = new VisualizationAPIComponent();
+  var visualizationAPIComponent = new VisualizationAPIComponent();
```

Comment:
This test was likely (wasn't working after upgrade at least) broken. This overwrote the constructor variable with a particular instance.